### DISCUSSION TITLE
Integration von HM-IP Broll fehlerhaft?

### DISCUSSION THREAD

**Discussion by JensHowi** [SukramJ/aiohomematic#1329] [Q&A]
Hallo zusammen, 
ich bin neu in der Materia HomeAssistant, habe bisher nur die Raspberrymatic auf einem Pi3 betrieben. Damit das ein oder andere vielleicht ein wenig für die anderen Mitbewohner visualisiert werden kann, habe ich mir mal auf einer Windows-VM HA als "Testinanz" aufgebaut.

Mit als letztes folgte bei mir die Integration der Raspberrymatic über "HomematicIP_Local". Alle Aktoren werden dort soweit korrekt angezeigt. Probleme macht mir jedoch der HMIP Broll, welchen ich überall als Rollladenaktor verbaut habe.

Zum einen bekomme ich dort nur Ereignisse und Diagnose-Daten angezeigt und beim Starten von HA bekomme ich noch eine Fehlermeldung. Aktuell ist es mir nicht möglich, die Rollläden über HA zu steuern. Was mache ich noch falsch, bzw. was fehlt mir?
Vielleicht könnt ihr mir helfen. Danke
![Screenshot 2023-12-17 150808](https://github.com/danielperna84/hahomematic/assets/153711552/b5485634-c8a1-4ebc-aa80-4f3bfa4ea7bf)
![Screenshot 2023-12-17 150922](https://github.com/danielperna84/hahomematic/assets/153711552/661de242-573d-435f-bdbb-4a436759dfdd)

**Comment by SukramJ** [comment]
Da Du eine Diskussion und kein Issue angelegt hast kann ich dein Frage nur mit "Der BROLL läuft super" beantworten.

Wenn Du jedoch wissen willst warum es bei dir nicht klappt kannst Du entweder ein Issue anlegen und die Diskussion hier schliessen,
oder alle Punkte, die im Issue abgefragt werden hier ergänzen.
